ui/pages/confirmations/utils/confirm.ts

```diff
@@ -1,13 +1,23 @@
 import { ApprovalRequest } from '@metamask/approval-controller';
 import { ApprovalType } from '@metamask/controller-utils';
+import { TransactionType } from '@metamask/transaction-controller';
 import { Json } from '@metamask/utils';
 
-const SignatureApprovalTypes = [
+export const REDESIGN_APPROVAL_TYPES = [
+  ApprovalType.PersonalSign,
+  ApprovalType.EthSignTypedData,
+] as const;
```

Comment:
May be also add `Transaction` here

Reply:
hmm true since we just created a ticket to separate the Signature Redesign logic from the ENV variable https://github.com/MetaMask/metamask-extension/issues/24312

I think, to be safe, we can add this following the above ticket

Reply:
sgtm! updated [e904ef2](https://github.com/MetaMask/metamask-extension/pull/24182/commits/e904ef2893fec4a59db599cf4fd7caa862f42672)